addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -99,6 +99,14 @@ html, body {
     height: 100%;
 }
 
+pre {
+    border: $border-width solid $border-color;
```

Comment:
Mind that the border-color here in frontend is using `var(--o-border-color)` which is equal to: `color-mix(in srgb, currentColor 15%, transparent)`

So if you want a consistent $o-gray-300 between inline-code and codeblock you might want to set the `--o-border-color` here as well. 👼